### DISCUSSION TITLE
May I join team

### DISCUSSION THREAD

**Discussion by hipexscape** [ROM-builders/temporary#19729] [General]
My git username - acex69
My telegram - @acex124

My dt - https://github.com/ProjectElixir-Devices/device_xiaomi_lancelot (it's new)
https://github.com/acex69/mt6768-common

My successful build 
https://cirrus-ci.com/build/5873010272370688

**Comment by Apon77** [comment]
GitHub and telegram username/name mismatched. also no need to tag. I get notification without tag

**Reply by hipexscape** [reply]
I am really sorry for tag sir 

Sir I can't change my tg username cause the git username is already taken 

I have changed to acex123
